corpus/convert.py

```diff
@@ -41,23 +41,18 @@ def run(self):
         orth_c.load(self.reference_bliss_corpus.get_path())
 
         orths = {}
-        for r in orth_c.all_recordings():
-            for i in range(len(r.segments)):
-                orth = r.segments[i].orth
-                tag = r.segments[i].fullname()
-                orths[tag] = orth
+        for s in orth_c.segments():
+            orth = s.orth
+            tag = s.fullname()
+            orths[tag] = name
```

Comment:
is the change `orth` -> `name` intentional? I don't think `name` is defined.
```suggestion
            orths[tag] = orth
```
the request changes is w.r.t. this comment.